Package.swift

```diff
@@ -0,0 +1,17 @@
+// swift-tools-version:5.0
+import PackageDescription
+
+let package = Package(
+  name: "RxGRDB",
+  products: [
+    .library(name: "RxGRDB", targets: ["RxGRDB"])
+  ],
+  dependencies: [
+    .package(url: "https://github.com/groue/GRDB.swift", from: "4.0.1"),
```

Comment:
Can we use semantic versioning instead?

```suggestion
    .package(url: "https://github.com/groue/GRDB.swift", .upToNextMajor(from: "4.0.1")),
```